src/app/components/Grid/index.jsx

```diff
@@ -46,8 +102,275 @@ export const CPSPageGrid = ({ children, ...props }) => (
 );
 
+export const ArticlePageGrid = ({ as, ...props }) => (
+  <Grid
+    {...props}
+    forwardedAs={as}
+    enableGelGutters
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 6,
+      group4: 8,
+      group5: 20,
+    }}
+  />
+);
+
+export const GridWrapper = ({ as, ...props }) => (
+  <GelPageGrid
+    {...props}
+    forwardedAs={as}
+    enableGelGutters
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 6,
+      group4: 8,
+      group5: 20,
+    }}
+  />
+);
+
+export const GridItemConstrainedSmall = styled(({ as, dir, ...props }) => (
+  <Grid
+    {...props}
+    forwardedAs={as}
+    dir={dir}
+    item
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
+      group5: 5,
+    }}
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 4,
+      group3: 5,
+      group4: 4,
+      group5: 8,
+    }}
+  />
+))`
+  ${layoutGridItemSmall}
+`;
+
+export const GridItemConstrainedMedium = ({
+  as,
+  gridColumnStart,
+  gridSpan,
+  dir,
+  ...props
+}) => (
+  <Grid
+    {...props}
+    forwardedAs={as}
+    dir={dir}
+    item
+    margins={{
+      group0: true,
+      group1: true,
+      group2: true,
+      group3: true,
+      group4: false,
+      group5: false,
+    }}
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
+      group5: gridColumnStart,
+    }}
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 5,
+      group4: 5,
+      group5: gridSpan,
+    }}
+  />
+);
+
+export const GridItemConstrainedMediumNoMargin = ({
+  as,
+  dir,
+  gridColumnStart,
+  gridSpan,
+  ...props
+}) => (
+  <Grid
+    {...props}
+    forwardedAs={as}
+    dir={dir}
+    item
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
+      group5: gridColumnStart,
+    }}
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 5,
+      group4: 5,
+      group5: gridSpan,
+    }}
+  />
+);
+
+export const GridItemConstrainedLarge = ({ as, dir, ...props }) => (
+  <Grid
+    {...props}
+    dir={dir}
+    forwardedAs={as}
+    item
+    margins={{
+      group0: true,
+      group1: true,
+      group2: true,
+      group3: true,
+      group4: false,
+      group5: false,
+    }}
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
+      group5: 5,
+    }}
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 6,
+      group4: 6,
+      group5: 12,
+    }}
+  />
+);
+
+export const GridItemConstrainedLargeNoMargin = ({ as, dir, ...props }) => (
+  <Grid
+    {...props}
+    forwardedAs={as}
+    dir={dir}
+    item
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
```

Comment:
See earlier comment RE the start column at this breakpoint

```suggestion
      group4: 2,
```